wallet/test/Test/Pos/Wallet/Web/Methods/PaymentSpec.hs

```diff
@@ -62,13 +63,22 @@ deriving instance Eq CTx
 
 -- TODO remove HasCompileInfo when MonadWalletWebMode will be splitted.
 spec :: Spec
-spec = beforeAll_ setupTestLogging $
-    withCompileInfo $
-       withDefConfigurations $ \_ txpConfig _ ->
-       describe "Wallet.Web.Methods.Payment" $ modifyMaxSuccess (const 10) $ do
-    describe "newPaymentBatch" $ do
-        describe "Submitting a payment when restoring" (rejectPaymentIfRestoringSpec txpConfig)
-        describe "One payment" (oneNewPaymentBatchSpec txpConfig)
+spec = do
+    runWithMagic RequiresNoMagic
+    -- Not running with `RequiresMagic` until `NetworkMagic` logic
+    -- has been fully implemented.
+    -- runWithMagic RequiresMagic
```

Comment:
Do these tests fail? I'm curious what the cause is. Seems ok to leave some of these for "implement-networkmagic", but we lose some of the nice assurance of correctness.

Reply:
These tests fail, and extricating the logic which causes them to fail is difficult and time consuming. So we'll defer this to the next PR.